src/hooks/api/getters/useProject/useProject.ts

```diff
@@ -1,37 +1,29 @@
-import useSWR, { mutate, SWRConfiguration } from 'swr';
-import { useState, useEffect } from 'react';
+import useSWR, { SWRConfiguration } from 'swr';
+import { useMemo, useCallback } from 'react';
 import { getProjectFetcher } from './getProjectFetcher';
 import { IProject } from 'interfaces/project';
 import { fallbackProject } from './fallbackProject';
 import useSort from 'hooks/useSort';
 
 const useProject = (id: string, options: SWRConfiguration = {}) => {
     const { KEY, fetcher } = getProjectFetcher(id);
+    const { data, error, mutate } = useSWR<IProject>(KEY, fetcher, options);
     const [sort] = useSort();
 
-    const { data, error } = useSWR<IProject>(KEY, fetcher, options);
-    const [loading, setLoading] = useState(!error && !data);
+    const refetch = useCallback(() => {
```

Comment:
I don't think `useCallback` is necessary here, if you assign `refetch: mutate` when returning.

Reply:
Right, the purpose of `useCallback` here is to avoid changing the type of the returned refetch.

Maybe a change for later when the hook itself has a proper return type.